packages/fast-usdc/src/exos/transaction-feed.ts

```diff
@@ -21,22 +18,18 @@ const TransactionFeedKitI = harden({
   }),
 });
 
-/**
- * @param {Zone} zone
- */
-export const prepareTransactionFeedKit = zone => {
+export const prepareTransactionFeedKit = (zone: Zone) => {
   const kinds = zone.mapStore('Kinds');
   const makeDurablePublishKit = prepareDurablePublishKit(
     kinds,
     'Transaction Feed',
   );
-  /** @type {PublishKit<CctpTxEvidence>} */
-  const { publisher, subscriber } = makeDurablePublishKit();
+  const { publisher, subscriber } =
+    makeDurablePublishKit() as PublishKit<CctpTxEvidence>;
 
```

Comment:
Is this a substantive change in static type checking?

Does casting with `as` require assignment compatibility? I think `@type` does.

Reply:
Once that type is generic, yes

Reply:
Done now